Formula/m/mecab-unidic.rb

```diff
@@ -1,14 +1,14 @@
 class MecabUnidic < Formula
   desc "Morphological analyzer for MeCab"
-  homepage "https://osdn.net/projects/unidic/"
+  homepage "https://clrd.ninjal.ac.jp/unidic/en/"
   # Canonical: https://osdn.net/dl/unidic/unidic-mecab-2.1.2_src.zip
   url "https://dotsrc.dl.osdn.net/osdn/unidic/58338/unidic-mecab-2.1.2_src.zip"
   sha256 "6cce98269214ce7de6159f61a25ffc5b436375c098cc86d6aa98c0605cbf90d4"
   license any_of: ["GPL-2.0-only", "LGPL-2.1-only", "BSD-3-Clause"]
 
   livecheck do
-    url "https://osdn.net/projects/unidic/releases/"
-    regex(%r{value=.*?/rel/unidic/unidic-mecab/v?(\d+(?:\.\d+)+)["' >]}i)
+    url "https://clrd.ninjal.ac.jp/unidic/en/back_number_en.html"
+    regex(/href=.*?unidic-mecab[._-]v?(\d+(?:\.\d+)+)_src\.z/i)
```

Comment:
```suggestion
    regex(/href=.*?unidic-mecab[._-]v?(\d+(?:\.\d+)+)[._-]src\.zip/i)
```

A couple of tweaks:

* We often use `[._-]` (or a subset of it) for suffix delimiters, as we don't want the regex to fail to match if upstream switches from `_src` to `-src` in the future
* We typically only abbreviate the extension for tarballs (as there are so many variations)